src/bokeh/models/widgets/tables.py

```diff
@@ -858,6 +858,47 @@ def __init__(self, *args, **kwargs) -> None:
     The height of each row in pixels.
     """)
 
+    @staticmethod
+    def from_data(data, columns=None, formatters={}, **kwargs) -> DataTable:
+        """ Create a simple table from a pandas dataframe, dictionary or ColumnDataSource.
+
+        Args:
+            data (DataFrame or dict or ColumnDataSource) :
+                The data to create the table from. If the data is a dataframe or dictionary, it will be converted to a ColumnDataSource
```

Comment:
We do try to keep docstrings under ~80 char width when possible (we don't fuss about long links, etc though)
```suggestion
            data (DataFrame or dict or ColumnDataSource) :
                The data to create the table from. If the data is a dataframe
                or dictionary, it will be converted to a ColumnDataSource.
```